<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>879</width>
    <height>860</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FontConverter</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QPushButton" name="openFileButton">
              <property name="toolTip">
               <string>Open font or image</string>
              </property>
              <property name="text">
               <string>Open</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="preset">
              <property name="toolTip">
               <string>Preset</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="bitcount">
              <property name="toolTip">
               <string>Bit count</string>
              </property>
              <property name="currentIndex">
               <number>-1</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="bitorder">
              <property name="toolTip">
               <string>Bit order</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="includes">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>40</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Includes</string>
              </property>
              <property name="plainText">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="arraySyntax1">
              <property name="toolTip">
               <string>Array declaration syntax</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="arraySyntax2">
              <property name="toolTip">
               <string>Array declaration syntax</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="generateButton">
              <property name="toolTip">
               <string>Generate code</string>
              </property>
              <property name="text">
               <string>Generate</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QGroupBox" name="fontInfoBox">
                <property name="title">
                 <string>Font info</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_2">
                 <item row="2" column="2">
                  <widget class="QSpinBox" name="firstChar">
                   <property name="maximum">
                    <number>999</number>
                   </property>
                   <property name="value">
                    <number>32</number>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="3">
                  <widget class="QLabel" name="lFontLast">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="3">
                  <widget class="QLabel" name="lFontFirst">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item row="5" column="1">
                  <widget class="QLabel" name="label_13">
                   <property name="text">
                    <string>Chars used</string>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="1">
                  <widget class="QLabel" name="label_12">
                   <property name="text">
                    <string>Last char</string>
                   </property>
                  </widget>
                 </item>
                 <item row="6" column="1">
                  <widget class="QLabel" name="label_19">
                   <property name="text">
                    <string>Size in bytes</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <widget class="QLabel" name="label_11">
                   <property name="text">
                    <string>First char</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QLabel" name="label_22">
                   <property name="text">
                    <string>Size &amp; stretch</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLabel" name="label_21">
                   <property name="text">
                    <string>Name</string>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="2">
                  <widget class="QSpinBox" name="lastChar">
                   <property name="maximum">
                    <number>999</number>
                   </property>
                   <property name="value">
                    <number>126</number>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="2" colspan="2">
                  <widget class="QLabel" name="lFontName">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="2" colspan="2">
                  <widget class="QLabel" name="lFontSize">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item row="5" column="2" colspan="2">
                  <widget class="QLabel" name="lFontUsed">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item row="6" column="2" colspan="2">
                  <widget class="QLabel" name="lFontBytes">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="charInfoBox">
                <property name="title">
                 <string>Char info</string>
                </property>
                <layout class="QGridLayout" name="gridLayout">
                 <item row="0" column="2">
                  <widget class="QLabel" name="lCharIndex">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>Index</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>Char</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="1">
                  <widget class="QLabel" name="label_4">
                   <property name="text">
                    <string>Scaled</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="2">
                  <widget class="QLabel" name="lCharScaled">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item row="6" column="2">
                  <widget class="QSpinBox" name="charCustomWidth">
                   <property name="minimum">
                    <number>8</number>
                   </property>
                   <property name="maximum">
                    <number>999</number>
                   </property>
                   <property name="singleStep">
                    <number>8</number>
                   </property>
                  </widget>
                 </item>
                 <item row="6" column="1">
                  <widget class="QCheckBox" name="charCustomWidthEnb">
                   <property name="text">
                    <string>Custom width</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="2">
                  <widget class="QLabel" name="lCharDim">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <widget class="QLabel" name="label_7">
                   <property name="text">
                    <string>Dimensions</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="2">
                  <widget class="QLabel" name="lCharChar">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="1">
                  <widget class="QLabel" name="label_14">
                   <property name="text">
                    <string>Size in bytes</string>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="2">
                  <widget class="QLabel" name="lCharBytes">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item row="5" column="1">
                  <widget class="QCheckBox" name="charIncluded">
                   <property name="text">
                    <string>Included</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="imgInfoBox">
                <property name="title">
                 <string>Image info</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_3">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_6">
                   <property name="text">
                    <string>Name</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="1">
                  <widget class="QLabel" name="lImgBytes">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <widget class="QLabel" name="lImgScaled">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="0">
                  <widget class="QLabel" name="label_15">
                   <property name="text">
                    <string>Size in bytes</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QLabel" name="label_10">
                   <property name="text">
                    <string>Scaled</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QLabel" name="lImgDim">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_8">
                   <property name="text">
                    <string>Dimensions</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLabel" name="lImgName">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="0">
                  <widget class="QCheckBox" name="imgCustomWidthEnb">
                   <property name="text">
                    <string>Custom width</string>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="1">
                  <widget class="QSpinBox" name="imgCustomWidth">
                   <property name="minimum">
                    <number>8</number>
                   </property>
                   <property name="maximum">
                    <number>999</number>
                   </property>
                   <property name="singleStep">
                    <number>8</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="lThreshold">
                <property name="text">
                 <string>Width threshold</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="threshold">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>7</number>
                </property>
                <property name="value">
                 <number>4</number>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>0</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QListWidget" name="listWidget">
          <property name="iconSize">
           <size>
            <width>500</width>
            <height>500</height>
           </size>
          </property>
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="viewMode">
           <enum>QListView::IconMode</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Size</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="glcdWidth">
            <property name="minimum">
             <number>8</number>
            </property>
            <property name="maximum">
             <number>999</number>
            </property>
            <property name="value">
             <number>320</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="glcdHeight">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>999</number>
            </property>
            <property name="value">
             <number>240</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="setGlcdSizeButton">
            <property name="text">
             <string>Set</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Zoom</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="pixelSize">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="value">
             <number>2</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spaceSize">
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Cursor</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="cursorX">
            <property name="maximum">
             <number>999</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="cursorY">
            <property name="maximum">
             <number>999</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>Text</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="text"/>
          </item>
          <item>
           <widget class="QToolButton" name="printButton">
            <property name="text">
             <string>Print</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="clearButton">
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="fill">
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="fillButton">
            <property name="text">
             <string>Fill</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGraphicsView" name="glcdView">
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="glcdInfo">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>879</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
